src/Pokemon/ElectricPokemon.java

```diff
@@ -0,0 +1,112 @@
+package Pokemon;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class ElectricPokemon extends Pokemon implements Attacks {
+    private static final String type = "electric";
+    private final List<String> attacks = Arrays.asList("electroBall", "thunder", "thunderPunch", "voltTackle");
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, type);
+    }
+
+
+    @Override
+    public List<String> getAttacks() {
+        return attacks;
+    }
+
+    public void electroBall(Pokemon name, Pokemon enemy) {
+        System.out.println(name.getName() + " attacks " + enemy.getName() + " with electroBall");
+        switch (enemy.getType()) {
+            case "water":
+                enemy.setHp(enemy.getHp() - 50);
+                System.out.println(enemy.getName() + " loses 50 hp");
```

Comment:
Deze regel herhaal je nu in elke switch statement. Beter kun je na de switch een enkel statment maken "[name] loses [x] hp"